### DISCUSSION TITLE
How can I remove X icon in select column in Form builder?

### DISCUSSION THREAD

**Discussion by alisalehi1380** [filamentphp/filament#13155] [Help] [answered]
### Package

Form builder

### Package Version

v3.0.0

### How can we help you?

![photo_2024-06-08_22-43-51](https://github.com/filamentphp/filament/assets/111766206/2267e88c-07a1-4423-8ef6-dfc08a12e379)

my code : 
```php
Select::make('status')
                ->required()
                ->autoThreeStatusOptions() //macro
                ->autoLabel(), //macro
```

if I can't remove the X icon and functionality, in your opinion What is the need for that, when I set required() for the column?!
user clicking in the X icon -> become empty input -> send form -> receive error response hah!

**Comment by leandrocfe** [comment, marked as answer]
`->selectablePlaceholder(false)`
https://filamentphp.com/docs/3.x/forms/fields/select#disable-placeholder-selection
